src/modals/AddressOptionsModal.tsx

```diff
@@ -74,7 +72,7 @@ const AddressOptionsModal = ({ address, onClose }: AddressOptionsModal) => {
       : t`To remove this address from being the main address, you must set another one as main first.`
 
   return (
-    <div ref={divRef} tabIndex={0}>
```

Comment:
Could you show me the broken scrolling?

This div is needed for accessibility purposes

Reply:
It's possible it was addressed in https://github.com/alephium/desktop-wallet/pull/353

Reply:
It is shown in the video. The unexpected scrolling happens when opening the address options modal. Removing the div does not affect accessibility, I can still navigate with the Tab key in the modal. Am I missing sth?

Reply:
The focus moves on the title of the modal, is that not enough?

Reply:
Ah yeah, it is - forgot `CenteredModal` takes care of this now